src/main/java/core/selenium/webdrivers/FactoryBrowser.java

```diff
@@ -0,0 +1,26 @@
+package core.selenium.webdrivers;
+
+/**
+ * FactoryBrowser class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public class FactoryBrowser {
+
+    /**
+     * Gets Browser with the parameter nameBrowser.
+     *
+     * @param nameBrowser use to create a Browser object.
+     * @return a Browser object.
+     */
+    public static Browser getBrowser(String nameBrowser) {
+        Browser browser;
+        if ("firefox".equals(nameBrowser)) {
```

Comment:
if you have a time maybe you want implement more browser like Ege, etc.

Reply:
Okay when I implement strategy pattern I add new browser, now only I use enum, please.